README.md

```diff
@@ -10,12 +10,19 @@ Clone this repo and run `composer install`.
 
 This will install all dependencies, including Drupal core and the Simplytest distribution profile.
 
+### DDEV
 The run the following command to install the site locally:
 
 ```
 ddev si
 ```
 
+### Lando
+The run the following command to install the site locally:
+```
+lando drush si simplytest --existing-config --account-pass=admin --yes
+```
```

Comment:
The instructions should mention `lando start` before this command.

Even better would be to have the site install automatically when you run `lando start`. You can do this by adding a `lando drush` command right after `composer install` in `.lando.yml`.